src/cli/Cargo.toml

```diff
@@ -39,20 +40,24 @@ prettytable-rs = "0.8.0"
 static_assertions = "1.1.0"
 tempdir = "0.3.7"
 sysinfo = "0.16.1"
+atty = "0.2.14"
```

Comment:
I am ok to moving consumer extension back into CLI crate due to complexity of maintaining multiple crates.  However, we should isolate consumer related functionality under a feature flag.

Reply:
Sure. I'm trying to think of a better name than "consumer" to describe this handful of features. These include consume/produce/topic/partition, so "consumer" seems too specific. Something like "cluster client"? That might be to vague, and clash with the idea of fluvio-cluster/fluvio(client). Let me know if you have any naming ideas or whether we should just stick with "consumer"

Reply:
Yeah... it seems almost like we have a "data" or "record" client (what we call "consumer" today), and then the "admin" client.